doc/widgets/preprocesstext.rst

```diff
@@ -13,20 +13,89 @@ Signals
+	- `Treebank POS Tagger (MaxEnt) <http://web.mit.edu/6.863/www/fall2012/projects/writeups/max-entropy-nltk.pdf>`_ runs POS tagging with a trained Penn Treebank model.
+	- `Stanford POS Tagger <http://nlp.stanford.edu/software/tagger.shtml#Download>`_ runs a log-linear part-of-speech tagger designed by Toutanova et al. Please download it from the provided website and load it in Orange.
+8. Produce a report.
+9. If *Commit Automatically* is on, changes are communicated automatically. Alternatively press *Commit*.
+
+.. note:: **Preprocess Text** applies preprocessing steps in the order they are listed. This means it will first transform the text, then apply tokenization, POS tags, normalization, filtering and finally constructs n-grams based on given tokens. This is especially important for WordNet Lemmatizer since it requires POS tags for proper normalization.
+
+Useful Regular Expressions
+--------------------------
+
+Here are some useful regular expressions for quick filtering:
+
+==================== =================================================
+``\bword\b``		 matches exact word 							   
+``\w+`` 			 matches only words, no punctuation 			   
+``\b(B|b)\w+\b``	 matches words beginning with the letter b 	   
+``\w{4,}``		     matches words that are longer than 4 characters  
+``\b\w+(Y|y)\b``	 matches words ending with the letter y 		   
+==================== =================================================
+
+Examples
+--------
+
+In the first example we will observe the effects of preprocessing on our text. We are working with *bookexcerpts.tab* that we've loaded with :doc:`Corpus <corpus>` widget. We have connected **Preprocess Text** to **Corpus** and retained default preprocessing methods (lowercase, per-word tokenization and stopword removal). The only additional parameter we've added as outputting only the first 100 most frequent tokens. Then we connected **Preprocess Text** with :doc:`Word Cloud <wordcloud>` to observe words that are the most frequent in our text. Play around with different parameters, to see how they transform the output.
+
+.. figure:: images/Preprocess-Text-Example1.png
+
+The second example is slightly more complex. We first acquired our data with :doc:`Twitter <twitter>` widget. We quired the internet for tweets from users @HillaryClinton and @realDonaldTrump and got their tweets from the past two weeks, 242 in total.
+
+.. figure:: images/Preprocess-Text-Example2.png
 
-No example yet for this widget.
+In **Preprocess Text** there's *Tweet* tokenization available, which retains hashtags, emojis, mentions and so on. However, this tokenizer doesn't get rid of punctuation, thus we expanded the Regexp filtering with symbols that we wanted to get rid of. We ended up with word-only tokens, which we displayed in :doc:`Word Cloud <wordcloud>`. Then we created a schema for predicting author based on tweet content, which is explained in more details in the documentation for :doc:`Twitter <twitter>` widget.
```

Comment:
You're right. I think it indeed could be a default, but I can still add it to the documentation. Your call.